app/helpers/batch_edits_helper.rb

```diff
@@ -1,14 +0,0 @@
-# View Helpers for Hydra Batch Edit functionality
-module BatchEditsHelper
```

Comment:
It's much easier to approve smaller PRs, so if you can make this change against master without all the other changes in this PR that would be preferable.